ui/modal/text_input_modal.go

```diff
@@ -2,28 +2,34 @@ package modal
 
 import (
 	"fmt"
+	"image/color"
 
 	"gioui.org/layout"
+	"gioui.org/text"
 	"gioui.org/widget"
 
 	"github.com/planetdecred/godcr/ui/decredmaterial"
 	"github.com/planetdecred/godcr/ui/load"
+	"github.com/planetdecred/godcr/ui/values"
 )
 
 const TextInput = "text_input_modal"
 
 type TextInputModal struct {
 	*InfoModal
 
-	IsLoading bool
+	IsLoading     bool
+	ShowWarnInfo  bool
```

Comment:
Should probably be specific since the info isn't customizable to any text. 
```suggestion
	ShowAccountWarnInfo  bool
```